README.md

```diff
@@ -44,6 +44,20 @@ If the schema does not already exist and the `db-init-script` is not in the defa
 
 Where `/tmp/schema.sql` contains: `CREATE SCHEMA IF NOT EXISTS my_db;`
 
+Full list of configuration options:
+|Property|Required|Description|
+|----|----|----|
+|`housekeeping.expired-path-duration`|No|Time To Live (TTL) of legacy replica paths in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601#Durations) format: only days, hours, minutes and seconds can be specified in the expression.|
+|`housekeeping.schema-name`|Yes|Database schema name to use. Tables will be created or used (if already existing) using this schema. Default: 'housekeeping'|
+|`housekeeping.db-init-script`|No|Database init script to use. Default: 'classpath:/schema.sql'
+|`housekeeping.data-source.driver-class-name`|No|Java classname of the database JDBC driver.|
+|`housekeeping.data-source.url`|No|JDBC connection URL.|
+|`housekeeping.data-source.username`|No|database user with access to schema.|
+|`housekeeping.data-source.password`|No|database user's password.|
+|`housekeeping.fetch-legacy-replica-path-page-size`|No| Page size of database fetch, tune this if you run out of memory or if it the query seems too slow. Higher number will use more memory. Default: '500'|
```

Comment:
Higher number will use more memory -> The higher the number, the more memory is required